spec/openapi.yaml

```diff
@@ -1697,6 +1697,61 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/404response'
+  /v1/messaging/numbers/sender_address/addresses/{id}/reverify:
+    post:
+      tags:
```

Comment:
Should be      tags:
        - Source Address ?

Reply:
Updated